### PR TITLE
Profile up section

### PR DESCRIPTION
**Pull request by Pepko0**
Here is my all styles for Profile up Section

### CODE REVIEW

File: src/features/personDetails/styled.js [hunk 1]
```diff
@@ -88,7 +78,7 @@ export const CastCrewWrapper = styled.div`
 `;
 
 export const ForDesktop = styled.div`
-  @media (max-width: 1198px) {
+  @media(max-width: ${({theme}) => theme.breakpoints.disabled}px){
```
Comment (Arkadiusz0809) [inline src/features/personDetails/styled.js:81]: why disabled?

File: src/styles/Theme.js [hunk 2]
```diff
@@ -23,6 +23,7 @@ export const theme = {
     mobileMax: 767,
     tabletVerticalMax: 991,
     tabletHorizontalMax: 1199,
+    disabled: 1367.99,
```
Comment (Arkadiusz0809) [inline src/styles/Theme.js:26]: what you need disabled and desktopMini, which differ them by 0.01 px ?

**Review by Arkadiusz0809 — COMMENTED** [review]
Give a sign !

File: src/styles/Theme.js (hunk 2 above)
Comment (BoosterTech) [reply src/styles/Theme.js:26]: good question. I wonder too

**Comment by BoosterTech** [comment]
In general it looks good :) Answer questions in comments from Arek before merging!

**Review by BoosterTech — APPROVED** [review]
Gra gitara, hucza skrzypce!!

**Pull request merged**